lib/trino-parquet/src/main/java/io/trino/parquet/writer/PrimitiveColumnWriter.java

```diff
@@ -57,6 +57,8 @@ public class PrimitiveColumnWriter
         implements ColumnWriter
 {
     private static final int INSTANCE_SIZE = instanceSize(PrimitiveColumnWriter.class);
+    private static final int MINIMUM_OUTPUT_BUFFER_CHUNK_SIZE = 4 * 1024;
+    private static final int MAXIMUM_OUTPUT_BUFFER_CHUNK_SIZE = 1024 * 1024;
```

Comment:
any specific reasons for those constants? maybe comment?

Reply:
This is copied from ORC writer
I think the exact numbers don't matter that much here, just want to avoid too big byte arrays and too small chunks for writing to output stream